example/App.js

```diff
@@ -72,4 +91,8 @@ const styles = StyleSheet.create({
     fontSize: 20,
     fontWeight: 'bold',
   },
+  dateTimeText: {
+    fontSize: 16,
+    fontWeight: 'normal',
+  },
```

Comment:
Remove extra `,` -- but that also means that a linter didn't run, so can you run the linter over these files?

Reply:
I have included the `example` folder for linting and it is requiring that `,`. I guess that this is how it is defined in `@react-native-community`.

Reply:
This is part of the react-native-community eslint config, they like their comma's :)